api.ts

```diff
@@ -1222,6 +1225,10 @@ export type MetadataFunction = (
   formulaContext?: MetadataContext,
 ) => Promise<MetadataFormulaResultType | MetadataFormulaResultType[] | ArraySchema | ObjectSchema<any, any>>;
 
+export type CellAutocompleteMetadataFunction = (
+  context: CellAutocompleteExecutionContext,
+) => Promise<MetadataFormulaResultType | MetadataFormulaResultType[] | ArraySchema | ObjectSchema<any, any>>;
```

Comment:
We can probably tighten this up--these functions don't need to return anything schema-related.

Reply:
Done